src/components/CodeView/index.tsx

```diff
@@ -182,7 +191,7 @@ export class CodeViewBase extends React.Component<Props> {
                             </td>
 
                             <td className={styles.code}>
-                              {renderHighlightedCode(code, language)}
+                              {renderCode(code, language, !codeWasTrimmed)}
```

Comment:
Could we switch to a single-object parameter for `renderCode()`? That would help understanding what the meaning of `!codeWasTrimmed` is.